src/main/java/at/ac/tuwien/ifs/sge/leeroy/agents/AttackActionSupplier.java

```diff
@@ -10,24 +10,24 @@
 
     private static final double RISK_THRESHOLD = 0.5;
 
-    public static Set<RiskAction> createActions(Risk risk) {
+    public static Set<RiskAction> createActions(Risk risk, Integer maxAttackerCnt) {
```

Comment:
What is `maxAttackerCnt` for?

Reply:
For the greedy leeroy attacker the supplier creates 'multi-round' actions, i.e. merges multiple attack phases into one (depending on game-config max n troops may attack). This is done to always attack with the largest army first, so large armies don't get stuck behind the frontline.

For the MCTS-Attack-Agent we only want valid actions, therefor the parameter